### PR TITLE
Add message info when LMD or Livestatus are down

### PR DESCRIPTION
**Pull request by avillalbaitrs**
If LMD or Livestatus are down only a message saying "Page has no content"
is shown. This is not too much information for the user.

If this commit is applied, the user will know which service is not running.

This fixes MONUI-76

Signed-off-by: Adrián Villalba <avillalba@itrsgroup.com>

### CODE REVIEW

File: application/views/css/layout.css [hunk 1]
```diff
@@ -1185,6 +1185,41 @@ h1.emote b {
   color: #fff;
 }
 
+<<<<<<< HEAD
+=======
```
Comment (webdevelin) [inline application/views/css/layout.css]: Seems to have been a merge conflict. These 2 lines and the one at the bottom should be removed. Row 1188, 1189 and 1222

File: application/views/template.php [hunk 2]
```diff
@@ -45,12 +45,21 @@
 								echo $content;
 							}
 						} else {
-							echo 'Page does not have any content';
-						}
+							$services_down = 'services LMD and Naemon are not running';
 
+							// Check LMD.
+							exec('service lmd status', $output, $return_var_lmd);
```
Comment (jacobbaungard) [inline application/views/template.php]: This will always return on non zero on 7.5. Doesn't make sense to do this check in 7.5 as we do not have LMD.

Comment (avillalbaitrs) [reply application/views/template.php]: So should we remove this branch?

Comment (jacobbaungard) [reply application/views/template.php]: On Monitor 7.5 I guess it is safe to say that when this page shows, it means that the Naemon service is not alive. So we can probably just write that directly without further checking.

Comment (avillalbaitrs) [reply application/views/template.php]: The "problem" here is that page is only shown when LMD is down but if both LMD and Naemon are down, the banner will say both are down. (At least in Monitor 8)

Comment (jacobbaungard) [reply application/views/template.php]: LMD isn't included in Monitor 7 at all.

**Pull request closed without merging**